### ISSUE TITLE
Wind is not changing with regular frequency

### ISSUE THREAD

**Issue by cambragol** [cambragol/advent-of-the-trinity#306]
**Describe the bug**
The wind appears to be stuck in one direction for long periods of time (more than a hundred moves)

**Comment by cambragol** [comment]
An easy tweak to change the frequency. Not sure what caused it but the fix is easy. This may represent a chance to tweak a default setting a bit, as I have seen multiple play throughs with complaints about how fast the wind changes, and I remember being frustrated with that as well, particularly when using the wind spell with the balloon.

**Comment by cambragol** [comment]
Of course, this all relates to how fast the CPU settings are. As I recall, @Fenyx4 played at a very low setting, compared to me, so perceptions of speed would vary.

**Comment by sethgupton-mastery** [comment]
Yeah. I play at 500 quite often because that is the default that comes with GOG.

**Comment by Fenyx4** [comment]
Yeah I use 500 quite often cuz that is the default that GOG uses and what is required for speedrunning.

**Comment by cambragol** [comment]
Is the wind change frequency really low in that situation?
